charts/k8tz/values.yaml

```diff
@@ -31,6 +31,12 @@ webhook:
 
   caBundle: |
 
+  # Minimum TLS version supported. Possible values: VersionTLS10, VersionTLS11, VersionTLS12, VersionTLS13
+  # If omitted, the default VersionTLS12 will be used.
+  tlsMinVersion: ""
+  # Comma-separated list of cipher suites for the server. If omitted, the default Go cipher suites will be used.
+  tlsCipherSuites: ""
```

Comment:
@klzsysy we need to move the field comment into https://github.com/k8tz/k8tz/blob/master/charts/k8tz/README.md#values

Reply:
done